bundles/asdi/asdi-projection-change/plugin/ProjectionChangerPlugin.js

```diff
@@ -0,0 +1,82 @@
+Oskari.clazz.define( 'Oskari.projection.change.ProjectionChangerPlugin',
+  function ( config, localization ) {
+    this._config = config || {};
+    this._clazz = 'Oskari.projection.change.ProjectionChangerPlugin';
+    this._defaultLocation = 'top right';
+    this._index = 55;
+    this._templates = {
+      projectionchanger: jQuery('<div class="mapplugin oskari-projection-changer"></div>')
+    };
+    this._loc = localization;
+
+    this._flyout = Oskari.clazz.create('Oskari.projection.change.flyout', this._loc, {
+        width: 'auto',
+        cls: 'projection-change-flyout'
+    });
+    this._flyout.makeDraggable({
+        handle : '.oskari-flyouttoolbar, .projection-data-container > .header',
```

Comment:
Probably don't need to send any params to makeDraggable. Or is drag-handle something more than the usual?